airbyte-integrations/connectors/source-tweag-blog/integration_tests/sample_config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "fix-me": "TODO"
```

Comment:
Still todo?

Reply:
Removed in https://github.com/tweag/airbyte/pull/2/commits/ad3cd0bf4105637948152fc7dccdf39c5ca5e6b5